### DISCUSSION TITLE
What is snesCIC?

### DISCUSSION THREAD

**Discussion by Jingleboy** [sanni/cartreader#384] [Need help]
Hey - love the project and hoping to build my own.

I'm reading through the build guide to understand everything, though this page about SA-1 SNES carts has left me confused: https://github.com/sanni/cartreader/wiki/Flashing-the-snesCIC

It states we have to flash snesCIC to a chip - however I cannot find any more information about "snesCIC". (The link on the page leads to sd2snes, which I'm unsure what it is)

Is this supposed to be superCIC? Or is snesCIC something else? (and I've missed something)

Also, if I happen to have the original CIC chip from my SNES, can that be used instead?

Apologies for the basic question, and thank you.

**Comment by sanni** [comment]
I didn't have time to design this part of the build yet. But the idea is to make a small PCB where you can solder either the superCIC, snesCIC(the smaller brother of the superCIC) or an original CIC from a cartridge or a console. That PCB then plugs into the header on the six slot adapter.

**Reply by Jingleboy** [reply]
Ah, good to know - I was getting ahead of myself.

I recently performed the superCNC mod to my SNES so I have an OG CIC chip laying around - it'd be great if I could recycle in the Cartreader!

Thanks for the response, and your work.

**Comment by Adalbertodfd** [comment]
where to get the .hex file for writing in 12f629?  can you send me the link to download the .hex file?

**Reply by sanni** [reply]
https://raw.githubusercontent.com/mrehkopf/sd2snes/develop/cic/snescic-lock-resync.hex
